components/builder-web/app/origin/origin-page/origin-packages-tab/origin-packages-tab.component.html

```diff
@@ -8,32 +8,26 @@
         (saved)="saved($event)">
       </hab-project-settings>
     </section>
+
     <section *ngIf="!selectingPlan">
-      <ul class="nav-list" *ngIf="projectsExist">
-        <li class="heading">
-          <h4>Connected Plans</h4>
-        </li>
-        <li class="item" *ngFor="let proj of projects">
-          <a [routerLink]="['/pkgs', origin, proj, 'latest']">
-            <span class="column name">{{ proj }}</span>
-          </a>
-          <hab-icon symbol="chevron-right"></hab-icon>
-        </li>
-      </ul>
-      <ul class="nav-list">
-        <li class="heading">
-          <h4>Packages</h4>
-        </li>
-        <li class="none" *ngIf="noPackages">
-          <span>No packages found.</span>
-        </li>
-        <li class="item" *ngFor="let pkg of packages">
-          <a [routerLink]="['/pkgs', pkg.origin, pkg.name, 'latest']">
-            <span class="column name">{{ pkg.name }}</span>
-          </a>
-          <hab-icon symbol="chevron-right"></hab-icon>
-        </li>
-      </ul>
+      <table>
```

Comment:
In it's current state, this would do better as a list element, however its matches the other views and will very likely go back to being a true table soon, so I feel we should leave it as a table.